test/header_parser_tests/expected_bindings/_expected_objc_interface_bindings.dart

```diff
@@ -180,41 +180,43 @@ class NativeLibrary {
       ffi.Pointer<ObjCObject> Function(ffi.Pointer<ObjCObject>,
           ffi.Pointer<ObjCSel>, ffi.Pointer<_NSZone>)>();
 
-  int _objc_msgSend_7(
+  bool _objc_msgSend_7(
     ffi.Pointer<ObjCObject> obj,
     ffi.Pointer<ObjCSel> sel,
     ffi.Pointer<ObjCSel> aSelector,
   ) {
     return __objc_msgSend_7(
-      obj,
-      sel,
-      aSelector,
-    );
+          obj,
+          sel,
+          aSelector,
+        ) !=
+        0;
```

Comment:
We do have a `Bool` type in `dart:ffi`, can we use that instead?

Reply:
I'm using the existing bool infrastructure for functions and structs. The code being generated here is consistent with existing cases:
https://github.com/dart-lang/ffigen/blob/master/test/code_generator_tests/expected_bindings/_expected_boolean_dartbool_bindings.dart

If you think this should generate Bool instead, that should probably be in a separate bug/PR. It would involve changing func.dart and compound.dart, and isn't really related to this PR.

Reply:
Agreed, that should be a separate PR if we don't target bool already.